### DISCUSSION TITLE
Unit's into Libre apps or to Libreview

### DISCUSSION THREAD

**Discussion by master3395** [NightscoutFoundation/xDrip#2769] [Ideas]
Not sure if it is even possible to import xDrip+'s treatment values into the libre2 and libre3 apps.
But if there is a way, perhaps we can have a way if there is one, so we don't have to double input the treatment if we want to use Libreview up against our hospitals. Or if xDrip+ can send it to Libreview or to [glooko.com,](https://glooko.com) since that is taking over diasend, as some hospitals use that.

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
It's been decided that xDrip+, not being an approved medical app, will not upload into vendors apps (because it would be illegal and put the app at risk of DMCA). You can use Tidepool and Nightscout.

**Reply by master3395** [reply]
How would it risk DMCA, when Tidepool and Nightscout can do it?

**Reply by Navid200** [reply]
@master3395 Please clarify what you mean?
Are you suggesting that Nightscout or Tidepool can upload to a Libre app?

**Reply by master3395** [reply]
Hey @Navid200, Psonnera suggested that I use either Tidepool or Nightscout to send my data to vendors' apps. However, if I decide to use Juggluco (a third-party app), I can upload my data to Libreview.com and glooko.com, and Juggluco can also populate the Libre3 app.

But the issue for me is that I would have to use three different apps to complete this task, and I would prefer to use just two apps, such as Libre3 and xDrip+, to upload my data to the vendors instead.
I have searched around, and it does indeed look like Tidepool and Nightscout are compatible to send to both sites, but in a really complicated way, that I might not even think I have the time to set up.

Here are some links for downloading Juggluco:

http://jkaltes.byethost16.com/Juggluco/download.html
https://www.juggluco.nl/Juggluco/download.html
https://github.com/j-kaltes/Juggluco

**Reply by Navid200** [reply]
First, Patrick can explain what he really meant.

There are things we decide not to do not because we cannot do them.   But, rather because we don't consider it to be acceptable practice on our part.

I would also like to use an example.  Someone asks me "tell me how to use xDrip to upload to Clarity".  I tell them they can use Nightscout.
What I mean there is not to use Nightscout to upload to Clarity.  No, absolutely not.  What I mean is that whatever the reason is you need to upload to Clarity (creating a report for your doctor, allowing your med team to monitor your glucose), you can use Nightsocut for.

Now, if the person tells me, I need to upload to Clarity for insurance, then, I will tell them, sorry, I have no solution for you.

You know it is very easy to complicate life really badly.  We often offer alternatives.  It may not be exactly what you need.  But, at the end of it all, we need to control our blood glucose.  Who said there was only one way to do it?

**Reply by psonnera** [reply]
@master3395 saying Tidepool is complicated shows you didn't even take the time to look at it. The fact that Juggluco uploads to LibreView is none of our business and since it achieves your goal I'd consider this issue closed.
